### ISSUE TITLE
Do the stream controller methods work?

### ISSUE THREAD

**Issue by ghost** [XanClic/g1#1]
Hi, I'm working on a game in Lua (Love2D) and have been looking to get the steam controller working for said game... so I haven't had a whole lot of luck. Do your steamcontroller.(cpp/hpp) files work (on linux, mac and windows or ?)?

**Comment by XanClic** [comment]
Hi,

I certainly hope so! I haven't tested them in a while, though, and I won't be home for another couple of days. Maybe I have my Steam Controller where I'm going tomorrow (although without the wireless transmitter...), so I'll see whether I can test it then (only on Linux, though, as I don't have Windows on my laptop—and because of some personal issues, I may not be able to bring my Steam Controller home for quite a while).

Max

**Comment by ghost** [comment]
Okay... I compiled the game but it wouldn't start bc it couldn't find a
controller

-sent from my OnePlus X

**Comment by XanClic** [comment]
Hm, weird. Normally it would start still (that is just a warning), but that's not what you're after anyway. I'll try to try my luck tomorrow but I don't know why it wouldn't recognize the controller.

**Comment by ghost** [comment]
Okay thx, appreciate it! If I can get this to work I'll make sure to
publish the resulting lua library

-sent from my OnePlus X

**Comment by XanClic** [comment]
It seems to work for me still. Maybe something else is concurrently using the controller (e.g. steam itself) and thus interferes?

**Comment by ghost** [comment]
Nope... It's not even installed. Do you have any sort of specific
permissions set?

-sent from my OnePlus X
